Yoast/wordpress-seo
# Expand passive verbs list for Farsi

## Context



## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Improves the Farsi passive voice assessment by expanding the list of passive verb forms.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Enable the `FARSI_SUPPORT` feature by adding `define( 'YOAST_SEO_FARSI_SUPPORT', true );` in `basic-wordpress-config.php` file
* Set your website to Farsi (فارسى).
* Test this functionality in Yoast SEO Free.
* Create a new empty post and add these sentences to your post:
`.شیشه شکانده شده است`
`.کارمند کشته شده بود`
`.غذا رفته است`

* Check that all sentences are highlighted in the passive voice assessment.

* Add these sentences and check that that they do not get highlighted in the passive voice assessment:
`.آنها دوچرخه را تعمیر می کنند`
`.باران شروع شد`
* Disable the `FARSI_SUPPORT` feature by adding `define( 'YOAST_SEO_FARSI_SUPPORT', false );` in `basic-wordpress-config.php` file

* Save the changes in the post

* Reload the page

* Confirm that the Passive voice assessment doesn't appear in the metabox

### Test instructions for QA when the code is in the RC


* [] QA should use the same steps as above.


QA can test this PR by following these steps:

* Set your website to Farsi (فارسى).
* Test this functionality in Yoast SEO Free.
* Create a new empty post and add these sentences to your post (note: you should copy/paste without formatting and then make sure that the full stops are still in the right place):
`.شیشه شکانده شده است`
`.کارمند کشته شده بود`
`.غذا رفته است`

* Check that all sentences are highlighted in the passive voice assessment.

* Add these sentences and check that that they do not get highlighted in the passive voice assessment:
`.آنها دوچرخه را تعمیر می کنند`
`.باران شروع شد`

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/secure/RapidBoard.jspa?rapidView=99&modal=detail&selectedIssue=LINGO-926
